### PR TITLE
fix papr runs

### PR DESCRIPTION
**Pull request by vrothberg**
Signed-off-by: Valentin Rothberg <vrothberg@suse.com>

### CODE REVIEW

**Comment by vrothberg** [comment]
@giuseppe PTAL. The recent hiccups from libpod are now here as well, but the change seems to fix it.

File: .papr.sh [hunk 1]
```diff
@@ -1,11 +1,10 @@
 #!/bin/bash
 set -xeuo pipefail
+export GOPATH=/go
+export PATH=$HOME/gopath/bin:$PATH:$GOPATH/bin
+export GOSRC=$GOPATH/src/github.com/containers/libpod
```
Comment (rhatdan) [inline .papr.sh]: libpod or buildah?

Comment (vrothberg) [reply .papr.sh]: I like both, but it should be buildah. Thanks for checking!

**Comment by giuseppe** [comment]
LGTM

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**